e2e_test/source/pubsub.slt

```diff
@@ -0,0 +1,82 @@
+# fail with invalid emulator_host
+statement error
+CREATE MATERIALIZED SOURCE s1 (v1 int, v2 varchar) WITH (
+    pubsub.subscription = 'test-subscription-1',
+    pubsub.emulator_host = 'localhost:5981',
```

Comment:
Will there be a formal key for the production environment?

Reply:
To test with production pubsub in CI?

Reply:
Yes, we are not hosting pubsub in CI env.

Reply:
I don't understand